dpub-aam/dpub-aam.html

```diff
@@ -437,7 +437,7 @@
                                                                 </td>
                                                                 <td><p>Expose <code>ROLE_SECTION</code> and object attribute <code>xml-roles:doc-colophon</code>. </p></td>
                                                                 <td>AXRole: <code>AXGroup</code><br />
-                                                                        AXSubrole: <code>&lt;nil&gt;</code><br/>
+                                                                        AXSubrole: AXApplicationGroup</code><br/>
```

Comment:
That's doc-colophon. I agree about the role and diff thing, but you can tell by the object attribute a couple of lines up. (IA2 and ATK require the actual/original role exposed in that fashion)